zip-xxxx.rst

```diff
@@ -0,0 +1,290 @@
+| **Common Transaction Fields**                                                                                                                                         |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``header``                |``uint32``                              |Contains:                                                            |
+|                             |                          |                                        |  * ``fOverwintered`` flag (bit 31, always set)                      |
+|                             |                          |                                        |  * ``version`` (bits 30 .. 0) – transaction version.                |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``nVersionGroupId``       |``uint32``                              |Version group ID (nonzero).                                          |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``nConsensusBranchId``    |``uint32``                              |Consensus branch ID (nonzero).                                       |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``lock_time``             |``uint32``                              |Unix-epoch UTC time or block height, encoded as in Bitcoin.          |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``nExpiryHeight``         |``uint32``                              |A block height in the range {1 .. 499999999} after which             |
+|                             |                          |                                        |the transaction will expire, or 0 to disable expiry.                 |
+|                             |                          |                                        |[ZIP-203]                                                            |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``8``                        |``fee``                   |``int64``                               |The fee to be paid by this transaction, in zatoshis.                 |
```

Comment:
Is it worth making this a CompactSize to save 5+ bytes per block?

Reply:
I have no strong opinion about this, but a possible argument for doing so is that if we increase the range of amounts in order to accommodate ZSA bridges with other coins that have a larger range, then using CompactSizes would help to mitigate the cost of that (when the full range is not actually used, or for ZEC).